contrib/cray/bin/run_libfabric_pipeline

```diff
@@ -172,6 +180,7 @@ export MPICH_PATH="${ROOT_BUILD_PATH}/mpich/stable"
 export SFT_INSTALL_PATH="${ROOT_BUILD_PATH}/libfabric-sft/stable"
 export BATS_INSTALL_PATH="${ROOT_BUILD_PATH}/bats/stable/bin"
 export BATS_LOG_DIRECTORY="$WORKSPACE/logs"
+export IMB_BUILD_PATH="${ROOT_BUILD_PATH}/imb/v2019.6"
```

Comment:
Replace v2019.6 with default

Reply:
It might be a good idea to leave it verbatim so we know what it is testing against though. What do you think?

Reply:
Lack of time probably.

Reply:
Feel free to change the default link.